dm/openapi/task.go

```diff
@@ -30,3 +32,13 @@ func (t *Task) Adjust() error {
 	}
 	return nil
 }
+
+// FromJSON unmarshal json to task.
+func (t *Task) FromJSON(data []byte) error {
+	return json.Unmarshal(data, t)
```

Comment:
maybe we can add those two func for all generated struct

Reply:
created an issue in https://github.com/deepmap/oapi-codegen/issues/489

Reply:
don't like the idea of adding methods to generated code, maybe just call Unmarshal/Marshal directly, it's just a wrapper

`task.Adjust()` is already merged, hate it too.

Reply:
add `json/toml/yaml` wrapper for generated struct such as `Source/Task/RelayConfig` is useful when we need convert between these types, and i think it adds to the readability of the code 😬
btw, we don't need to add those func manually, they can be generated automatilly via `oapi`

`task.Adjust()` is same with many other `AdjustXX` func in our codebase, can you explain why you think this is bad?

Reply:
it's fine if it can be generated automatically.

`task.Adjust()` is enhancing a generated type, not a good practice i think.